src/Tizen.TV.UIControls.Forms/Renderer/UIControlsExtensions.cs

```diff
@@ -53,6 +52,9 @@ public static MauiAppBuilder UseTizenTVUIControls(this MauiAppBuilder builder)
 
                     //TODO
                     //handlers.AddCompatibilityRenderer(typeof(AnimatedNavigationPage), typeof(AnimatedNavigationPageRenderer));
+
+                    // This is a workaround for wrong ListView behavior
+                    handlers.AddHandler<ListView, TVListViewRenderer>();
```

Comment:
I wonder why didn't you use `AddCompatibilityRenderer`?

Reply:
Because renderer did not inherit the `IRegisterable` interface.